src/Tools/Source/CompilerGeneratorTools/CompilerTools.sln

```diff
@@ -1,17 +1,19 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
```

Comment:
I am assuming changes to this file were not intentional. Please revert/exclude from the PR.

Reply:
There were intentional. The CompilerTools.sln didn't have the new CompilersIOperationGenerator added to it.